libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/utils/UiAutomationUtils.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.navigation.testing.ui.utils
+
+import android.app.UiAutomation
+import java.io.FileInputStream
+import java.io.IOException
+import java.io.InputStream
+
+fun UiAutomation.executeShellCommandBlocking(command: String) {
+    val output = executeShellCommand(command)
+
+    val stream: InputStream = FileInputStream(output.fileDescriptor)
+    try {
+        val buffer = ByteArray(1024)
+        while (stream.read(buffer) != -1) {
+            // Consume stdout to ensure the command completes
+        }
+    } catch (ignored: IOException) {
+    } finally {
+        try {
+            stream.close()
+        } catch (ignored: IOException) {
+        }
+        try {
+            output.close()
+        } catch (ignored: IOException) {
+        }
+    }
+}
```

Comment:
I think we can make this function more useful and simple this way. 
Also we shouldn't wrap this with try-catch, because in tests we want to fail early if something went wrong. 
```suggestion
fun UiAutomation.executeShellCommandBlocking(command: String): ByteArray {
    val output = executeShellCommand(command)
    return FileInputStream(output.fileDescriptor).use { it.readBytes() }
}
```

Reply:
Thanks for suggestion, it does look better. But I don't have 100% confidence that it's going to work in the same way. @Zayankovsky , are you ready to spend time scheduling another 10 runs on CI?

Reply:
You can just push 10 branches with different commit hashes, but same contents and open 10 draft PRs :)
Then all these runs will be done in parallel.

Reply:
@Zayankovsky , are you ready to help me with testing? 😉

Reply:
Sure, what can I do?

Reply:
Commit the suggestion and schedule `instrumentation-tests` to run 10 times.

Reply:
Here you go, all successful:
#5193 #5194 #5195 #5196 #5197 #5198 #5199 #5200 #5201 #5202

Reply:
@Zayankovsky , thanks a lot for the help ❤️ 
I will apply fixes and cleanup PRs.